packages/new-deploy/src/bridge/BridgeContracts.ts

```diff
@@ -524,25 +524,23 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
     const core = this.context.mustGetCore(this.domain);
     const governance = core.governanceRouter.address;
 
-    const contracts = [this.tokenRegistryContract, this.bridgeRouterContract];
     const deployer = await this.deployer.getAddress();
 
-    // conditional to avoid erroring
-    const txns = await Promise.all(
-      contracts.map(async (contract) => {
-        const owner = await contract.owner();
-        if (utils.equalIds(owner, deployer)) {
-          log(`transfer bridge ownership on ${name}`);
-          return await contract.transferOwnership(governance, this.overrides);
-        }
-      }),
-    );
+    // transfer ownership of tokenRegistry to bridge
+    const registryOwner = await this.tokenRegistryContract.owner();
+    if (utils.equalIds(registryOwner, deployer)) {
```

Comment:
is this pattern used everywhere? it fails if `deployer` is equal to (e.g.) the governance because we have used a safe-ethers-signer. File an issue to correct this.

correct pattern would be `if (!utils.equalIds(registryOwner, correctOwner)))`